cli/cli.go

```diff
@@ -1398,3 +1396,17 @@ func readForConfirmationBlockUntilValid(r io.Reader, prompt string) bool {
 		fmt.Println(InputParsingError)
 	}
 }
+
+func composeAppVersion(buildVersion string, environment string, isSnap bool) string {
```

Comment:
We have `config.BuildTarget` that contains version and environment. Not sure if it makes sense to reuse here and also it does not have information if this is snap build currently, so maybe doesn't make sense to use it, but leaving a comment here just FYI